lib/dogma/util/script_strings.ex

```diff
@@ -7,98 +7,106 @@ defmodule Dogma.Util.ScriptStrings do
   for Elixir code and reporting errors when there should be none.
   """
 
+  @sigil_delimiters [{"|", "|"}, {~S("), ~S(")}, {"'", "'"}, {"(", ")"},
+                      {"[", "]"}, {"{", "}"}, {"<", ">"}]
+  @all_string_sigils @sigil_delimiters
+                      |> Enum.flat_map(fn({b, e}) ->
+                          [{"~s#{b}", e}, {"~S#{b}", e}]
```

Comment:
You're the maintainer, it is your choice (and responsibility) to decide what goes into the codebase and what doesn't. Me and the other contributors, like @5thWall and @alxndr, are simply trying to add something of value.

That said, `Dogma.Util.ScriptStrings` is broken at the moment. It does simply not do what it promises to do (again "take a source string and return it with all of the string literals stripped of their contents."). This PR fixes it. If you don't want it, because you realized you did want that specific functionality in the first place, that is completely fine.

But "we might change this code in the future to achieve something not described today" is not a valid reason to reject a PR that fixes a current problem, IMHO.